kfdefs/base/monitoring/blackbox-exporter/prometheus.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
```

Comment:
@hemajv is there a reason we are adding a new Prometheus instance and not using the existing one?